app/src/main/java/com/kickstarter/libs/KSLifecycleEvent.kt

```diff
@@ -0,0 +1,5 @@
+package com.kickstarter.libs
+
+enum class KSLifecycleEvent {
```

Comment:
This enum class replaces the FragmentEvent class from Trello's RxLifecycle library